components/apimgt/org.wso2.carbon.apimgt.rest.api.publisher.v1/src/main/java/org/wso2/carbon/apimgt/rest/api/publisher/v1/impl/ApiProductsApiServiceImpl.java

```diff
@@ -639,7 +639,7 @@ public Response apiProductsGet(Integer limit, Integer offset, String query, Stri
 
         try {
             //for now one criterea is supported
-            String searchQuery = StringUtils.replace(query, ":", "=");
+            String searchQuery = RestApiConstants.GET_API_PRODUCT_QUERY;
```

Comment:
We should allow passing custom queries to api-products GET resource. This modification will fetch all the API Products every time, disregarding the query being passed.

Reply:
Fixed